specifyweb/specify/migrations/0002_geo.py

```diff
@@ -0,0 +1,168 @@
+                ('timestampcreated', models.DateTimeField(db_column='TimestampCreated', default=django.utils.timezone.now)),
+                ('timestampmodified', models.DateTimeField(blank=True, db_column='TimestampModified', default=django.utils.timezone.now, null=True)),
+                ('createdbyagent', models.ForeignKey(db_column='CreatedByAgentID', null=True, on_delete=specifyweb.specify.models.protect_with_blockers, related_name='+', to='specify.agent')),
+                ('modifiedbyagent', models.ForeignKey(db_column='ModifiedByAgentID', null=True, on_delete=specifyweb.specify.models.protect_with_blockers, related_name='+', to='specify.agent')),
+            ],
+            options={
+                'db_table': 'collectionobjectgrouptype',
+                'ordering': (),
+            },
+        ),
+        migrations.AddField(
+            model_name='collectionobject',
+            name='cotype',
+            field=models.ForeignKey(db_column='COTypeID', null=True, on_delete=models.SET_NULL, related_name='collectionobjects', to='specify.collectionobjecttype'),
+        ),
+        migrations.RunPython(create_default_collection_types, revert_default_collection_types),
+        migrations.CreateModel(
+            name='CollectionObjectGroup',
+            fields=[
+                ('id', models.AutoField(db_column='collectionobjectgroupid', primary_key=True, serialize=False)),
+                ('name', models.CharField(blank=True, db_column='Name', max_length=255, null=True)),
+                ('description', models.TextField(blank=True, db_column='Description', null=True)),
+                ('igsn', models.CharField(blank=True, db_column='IGSN', max_length=255, null=True)),
+                ('guid', models.CharField(blank=True, db_column='GUID', max_length=255, null=True)),
+                ('isconsolidated', models.BooleanField(blank=True, db_column='IsConsolidated', null=True)),
+                ('integer1', models.IntegerField(blank=True, db_column='Integer1', null=True)),
+                ('integer2', models.IntegerField(blank=True, db_column='Integer2', null=True)),
+                ('integer3', models.IntegerField(blank=True, db_column='Integer3', null=True)),
+                ('decimal1', models.DecimalField(blank=True, db_column='Decimal1', decimal_places=10, max_digits=22, null=True)),
+                ('decimal2', models.DecimalField(blank=True, db_column='Decimal2', decimal_places=10, max_digits=22, null=True)),
+                ('decimal3', models.DecimalField(blank=True, db_column='Decimal3', decimal_places=10, max_digits=22, null=True)),
+                ('text1', models.TextField(blank=True, db_column='Text1', null=True)),
+                ('text2', models.TextField(blank=True, db_column='Text2', null=True)),
+                ('text3', models.TextField(blank=True, db_column='Text3', null=True)),
+                ('boolean1', models.BooleanField(blank=True, db_column='Boolean1', null=True)),
```

Comment:
Boolean fields should be `YesNo1`, `YesNo2`, and `YesNo3` rather than `Boolean` to match our existing convention.